django_perf_rec/api.py

```diff
@@ -10,10 +10,12 @@
 
 from .cache import AllCacheRecorder
 from .db import AllDBRecorder
+from .settings import PerfRecSettings
 from .utils import current_test
 from .yaml import KVFile
 
 
+settings = PerfRecSettings()
```

Comment:
I think the singleton should live next to the class. It's a pattern I've used in e.g. https://github.com/YPlan/nexus/blob/master/nexus/conf.py